SwiftCSVTests/EnumeratedCSVViewTests.swift

```diff
@@ -84,4 +84,15 @@ class EnumeratedViewTests: XCTestCase {
         csv = try CSV<Enumerated>(string: "id,\"the, name\",age\n1,\"Alice, In, Wonderland\",18\n2,Bob,19\n3,Charlie,20")
         XCTAssertEqual(csv.serialized, "id,\"the, name\",age\n1,\"Alice, In, Wonderland\",18\n2,Bob,19\n3,Charlie,20")
     }
+
+    func testSerializationWithDifferentDelimiters() throws {
```

Comment:
Thanks for adding a test!

WDYT about combining all `testSerialization*` functions into one? The separation doesn't add any value, and splitting your 3 new test cases into 3 test functions like "testSerializationWithTabDelimierAndQuotes" gets tedious. 

You could then add a short comment before each `csv = try...` initialization to describe the case:

- comma-separated values
- comma-separated values with double quotes
- tab-separated values
- tab-separated values with commas in cells

Reply:
That sounds reasonable.